cli/utils/ts-linter.js

```diff
@@ -1,51 +1,62 @@
 /*jslint node: true */
 'use strict';
 
-const spawn = require('cross-spawn');
 const logger = require('@blackbaud/skyux-logger');
+const tslint = require('tslint');
 const skyPagesConfigUtil = require('../../config/sky-pages/sky-pages.config');
 
-const flags = [
-  '--project',
-  skyPagesConfigUtil.spaPath('tsconfig.json'),
-  '--config',
-  skyPagesConfigUtil.spaPath('tslint.json'),
-  '--exclude',
-  '**/node_modules/**/*.ts'
-];
+const { Configuration, Linter, Formatters } = tslint;
```

Comment:
Just my personal preference, but do you need the `tslint` variable anywhere? Could this be imported as:
```
const { Configuration, Linter, Formatters } = require('tslint);
```

Reply:
Adjusted.